config/src/test/java/org/springframework/security/config/annotation/web/configurers/RememberMeConfigurerTests.java

```diff
@@ -189,7 +191,8 @@ protected void configure(HttpSecurity http) throws Exception {
 		@Bean
 		public UserDetailsService userDetailsService() {
 			return new InMemoryUserDetailsManager(
-					User.withDefaultPasswordEncoder()
```

Comment:
Is this change necessary? `User.withDefaultPasswordEncoder` is deprecated, but it is acceptable to use in tests.

Reply:
IDEA told me there is a warning when I commit this file, so I changed it. Should I revert this change?

Reply:
Yes, please revert this change. 
The IDE is correct to show a warning, but in this specific case it is allowed.

Reply:
Done